mobile/src/reducers/calendar/dayoff.epics.ts

```diff
@@ -33,9 +33,9 @@ export const dayoffSavedEpic$ = (action$: ActionsObservable<ConfirmProcessDayoff
                 `/employees/${x.employeeId}/events`,
                 calendarEvents,
                 { 'Content-Type': 'application/json' }
-            ).map(obj => deserialize(obj.response, CalendarEvent));
+            ).map(obj => deserialize(obj.response, CalendarEvent))
+            .map(() => loadCalendarEvents(x.employeeId));
         })
-        .map(x => calendarEventCreated(x))
```

Comment:
Didn't you forget to delete the whole code?

Reply:
Thanks! Addressed with latest commit.